specs/plugins.md

```diff
@@ -0,0 +1,173 @@
+# Support a "plug-in" model to provide extensibility
+
+To allow developers to provide unique handling behaviors for different APIs the Microsoft Graph will have a plug-in model provide an extension point for developer. This will provide separation of concerns between the software components intercepting network request and those providing behaviors.
```

Comment:
```suggestion
To allow developers to provide unique handling behaviors for different APIs the Microsoft Graph Developer Proxy will have a plug-in model provide an extension point for developer. This will provide separation of concerns between the software components intercepting network request and those providing behaviors.
```